### DISCUSSION TITLE
VisIt SDF plugin problem

### DISCUSSION THREAD

**Discussion by xh-ui** [epochpic/epoch#670] [Q&A]
Dear all,

I built the VisIt SDF plugin in Ubuntu and a success message is also shown. However, the generated **".visit"** folder finally contains **three empty subfolders ("databases", "operators", and "plots")**. 

The Ubuntu is 22.04, VisIt is 3.4.1.

**Comment by rashid611** [comment]
in my case , i can only use Visit3.2.2. the higher versions didnt work for me. i suggest you to try 3.2.2.

**Reply by xh-ui** [reply]
thank you!   I'll try the 3.2.2 version.

**Reply by phy-ak** [reply]
> in my case , i can only use Visit3.2.2. the higher versions didnt work for me. i suggest you to try 3.2.2.

In my case, there is a error, libicui18n.so.67: cannot open share object file: No such file or directory. When i tried to install 3.2.2. in ubuntu 22 this is the error i am facing. Can you please help me over it.

**Reply by rashid611** [reply]
The error is because your system is missing a specific version of the ICU (International Components for Unicode) library. This can often happen if a software package was built against a version of ICU that is not available on your system.
I think you should try to install the icu library first,

```
sudo apt-get update
sudo apt-get install libicu67
```

**Reply by phy-ak** [reply]
I have already tried installin it. Its showing unable to locate package. since my icu version is libicu70 it seems. How you have corrected this error.

**Reply by rashid611** [reply]
If I am not wrong you have libicu70 installed and are encountering issues with locating libicui18n.so.67. 
Then creating symbolic links might be the best solution, 

```
cd /usr/lib/x86_64-linux-gnu/       #Navigate to the library directory

sudo ln -s libicui18n.so.70 libicui18n.so.67      #Create symbolic links for the required ICU libraries
sudo ln -s libicuuc.so.70 libicuuc.so.67
sudo ln -s libicudata.so.70 libicudata.so.67
```
